daemon/mgr/container.go

```diff
@@ -355,6 +355,18 @@ func (mgr *ContainerManager) Create(ctx context.Context, name string, config *ty
 		HostConfig: config.HostConfig,
 	}
 
+	// merge image's config into container
+	if err := container.merge(func() (ocispec.ImageConfig, error) {
+		img, err := mgr.Client.GetImage(ctx, config.Image)
```

Comment:
no error check?

Reply:
This should ask the one wrote for it, seems like a bug.